library/pk.c

```diff
@@ -151,7 +159,8 @@ int mbedtls_pk_setup_rsa_alt( mbedtls_pk_context *ctx, void * key,
     mbedtls_rsa_alt_context *rsa_alt;
     const mbedtls_pk_info_t *info = &mbedtls_rsa_alt_info;
 
-    if( ctx == NULL || ctx->pk_info != NULL )
+    PK_VALIDATE_RET( ctx != NULL );
```

Comment:
Does it make sense to use RSA ALT contexts with NULL key, e.g. when wrapping an HSM? In this case, I'm OK with not adding more checks, but the documentation should definitely mention the requirements here.

Reply:
I don't know, but in doubt, I prefer to not change the behavior.

Reply:
Ok.